python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -496,7 +496,7 @@ def _bulk_action_filters_from_run_filters(filters: RunsFilter) -> BulkActionsFil
         _bulk_action_statuses_from_run_statuses(filters.statuses) if filters.statuses else None
     )
     backfill_ids = None
-    if filters.tags and filters.tags.get(BACKFILL_ID_TAG) is not None:
+    if filters.tags.get(BACKFILL_ID_TAG) is not None:
```

Comment:
in making the change in this PR i noticed that the `filters.tags` check is unnecessary since `filters.tags` is never None, it defaults to `{}`